ios/Classes/NativeBridge/Handlers/RegistrationHandler.swift

```diff
@@ -1,11 +1,10 @@
 import OneginiSDKiOS
 
 protocol RegistrationConnectorToHandlerProtocol: RegistrationHandlerToPinHanlderProtocol {
-    func signUp(_ providerId: String?, scopes: [String]?, completion: @escaping (Bool, ONGUserProfile?, ONGCustomInfo?, SdkError?) -> Void)
-    func processRedirectURL(url: String, webSignInType: WebSignInType)
+    func registerUser(_ providerId: String?, scopes: [String]?, completion: @escaping (Result<OWRegistrationResponse, FlutterError>) -> Void)
+    func processRedirectURL(url: String, webSignInType: Int) -> Result<Void, FlutterError>
```

Comment:
Well, these functions are actually not async and I believe it's safer to do this. Because now we can not make the mistake of internally forgetting to call completion. What do you think?